Cargo.toml

```diff
@@ -4,3 +4,6 @@ members = [
   "cli",
   "xtask",
 ]
+
+[patch.crates-io]
```

Comment:
Might want to get the `rusty_v8` repo under `phylum-dev` at some point, if it's gonna be the long term solution!